tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language-subtag-registry

```diff
@@ -44795,7 +44795,7 @@ Added: 2005-10-16
 Type: script
 Subtag: Kawi
 Description: Kawi
-Added: 2021-12-22
+Added: 2021-12-24
```

Comment:
FYI  The old dates were my guesses from when I cherry-picked just the new records, after they were proposed but before they went live in the LSR :-)